<ui version="4.0" >
 <class>AboutWindow</class>
 <widget class="QDialog" name="AboutWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>@product@ - About</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="pixmap" >
      <pixmap resource="../../../resources/wengophone.qrc" >:/pics/about_logo.png</pixmap>
     </property>
     <property name="alignment" >
      <set>Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <widget class="QWidget" name="aboutTab" >
      <attribute name="title" >
       <string>About</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="4" column="0" >
        <widget class="QLabel" name="versionLabel" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="descriptionLabel" >
         <property name="text" >
          <string>@product@, a voice over Internet phone</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="copyrightLabel" >
         <property name="text" >
          <string>Copyright (C)2008 Mbdsys</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="licenseLabel" >
         <property name="text" >
          <string>This program is free software; you can redistribute it and/or
modify it under the terms of the GNU General Public License
as published by the Free Software Foundation; either version 2
of the License, or (at your option) any later version.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="authorsTab" >
      <attribute name="title" >
       <string>Authors</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>9</number>
       </property>
       <property name="topMargin" >
        <number>9</number>
       </property>
       <property name="rightMargin" >
        <number>9</number>
       </property>
       <property name="bottomMargin" >
        <number>9</number>
       </property>
       <property name="horizontalSpacing" >
        <number>6</number>
       </property>
       <property name="verticalSpacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTextEdit" name="authorsTextEdit" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="wengoPhoneBuildIdStringLabel" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>301</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>closeButton</tabstop>
  <tabstop>authorsTextEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resources/wengophone.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>AboutWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>453</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel" >
     <x>212</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
